src/de/cuuky/varo/game/world/setup/AutoSetup.java

```diff
@@ -128,4 +130,40 @@ private void setupSpawns(Location middle) {
             new AutoStart(start);
         }
     }
-}
\ No newline at end of file
+
+    private Location getLobbyLocation(World world, long x, long z) {
+        return new Location(world, x, getLobbyHeight(world, x, z), z);
+    }
+
+    private int getLobbyHeight(World world, long x, long z) {
+        int heightSetting = ConfigSetting.AUTOSETUP_LOBBY_SNAP_OFFSET.getValueAsInt();
+        LobbySnap snapSetting = LobbySnap.valueOf(ConfigSetting.AUTOSETUP_LOBBY_SNAP.getValueAsString());
```

Comment:
I think there should be some sort of error handling in case the user sets `AUTOSETUP_LOBBY_SNAP` to an invalid value.

Reply:
How exactly do you imagine this error handling? As far as I know there is no built-in method to validate config entries directly. If the code is executed and there is a wrong entry, I think there should be an exception, since it is a severe error. Such an error prevents the autosetup from completing and without a fallback to the default value, it is not possible to complete it. If I simply use the default value when an error occurs, confusion may arise because the error may not be detected and the behavior of the plugin may be different than expected.

Reply:
I could simply catch the exception in the #setupLobby method and then print an error message.

```java
            Location lobby;
            try {
                 lobby = getLobbyLocation(world.getWorld(), x, z);
            } catch (IllegalArgumentException ex){
                System.err.println(Main.getConsolePrefix() + "AutoSetup: The config value for lobby.snap is invalid!");
                ex.printStackTrace();
                return;
            }
```

Reply:
Not really a nice solution, but should be good enough.

Reply:
Yes, that's why I originally just didn't do any error-handling. If you have a better suggestion feel free to let me know.

Reply:
Bad error handling is still better than no error handling. The proper way to do this would be to have the config system validate the values when loading. But that would be a lot of work and the config system will be reworked in v5 anyway.